components/employee/RegisteredMyProfile/RegisteredMyProfile.tsx

```diff
@@ -0,0 +1,45 @@
+import { CommonBtn } from "components/common";
+import Image from "next/image";
+import Link from "next/link";
+import { ButtonStyle } from "types/enums/button.enum";
+import styles from "./RegisteredMyProfile.module.scss";
```

Comment:
```suggestion
import Image from "next/image";
import Link from "next/link";
import { CommonBtn } from "components/common";
import { ButtonStyle } from "types/enums/button.enum";
import styles from "./RegisteredMyProfile.module.scss";
```
import 순서 컨벤션 준수합시다!